components/Carousel.js

```diff
@@ -76,46 +73,46 @@ class Carousel {
       this._numberOfSlidesPerPage = 1;
     }
     const numOfSlides = carouselSlides.length;
-
     this._numberOfTransitions = Math.ceil(numOfSlides / this._numberOfSlidesPerPage) - 1;
-    this._slidePersentageOutOfScreen = (1 / this._numberOfSlidesPerPage) * 100;
-    const carouselOverflowWidth = Math.ceil(this._slidePersentageOutOfScreen * carouselSlides.length);
-    carouselTrack.style.width = carouselOverflowWidth + "%";
+    const slidePersentageOutOfScreen = (1 / this._numberOfSlidesPerPage) * 100;
+    const carouselOverflowWidth = Math.ceil(slidePersentageOutOfScreen * carouselSlides.length);
+
+    this._carouselTrack.style.width = carouselOverflowWidth + "%";
+
     const carouselSlidesStyle = getComputedStyle(carouselSlides[0]);
     this._sizeOfSlide = carouselSlides[0].clientWidth + parseInt(carouselSlidesStyle.getPropertyValue('margin-right'), 10);
   }
 
   _setEventListeners() {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
     const prevButton = this._carouselElement.querySelector('.carousel__slide-button_type_prev');
     const nextButton = this._carouselElement.querySelector('.carousel__slide-button_type_next');
     //Button Listeners
     nextButton.addEventListener('click', () => {
-      carouselTrack.style.animationPlayState = "paused";
+      this._carouselTrack.style.animationPlayState = "paused";
```

Comment:
These should be single quotes for consistency.

Reply:
done